engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/VariableInstanceQueryDto.ftl

```diff
@@ -0,0 +1,17 @@
+<@lib.dto
+    desc = "A variable instance query which defines a list of variable instances" >
+    <#assign requestMethod="POST"/>
+    <#include "/lib/commons/variable-instance-query-params.ftl" >
+    <@lib.properties params />
+
+    "sorting": {
+      "type": "array",
+      "description": "Apply sorting of the result",
```

Comment:
We can already add clarification that the property is not applicable for the POST count endpoint (CAM-12858).

Reply:
I will add it here, but I was also think about solving that ticket with a macro. There is quite a lot of duplication of the sorting json in many QueryDtos. Also it is not always correct. Here for example large parts of the description are missing, if you compare it to the docs.

Reply:
👍